vaccine_feed_ingest/runners/_shared/parse.py

```diff
@@ -60,7 +62,21 @@ def _output_ndjson(json_list: List[dict], out_filepath: pathlib.Path) -> None:
             fout.write("\n")
 
 
+def _prepmod_find_data_item(parent, label, offset):
+    try:
+        row_matches = [
+            x for x in parent.find_all(["p", "div"]) if label in x.get_text()
+        ]
+        content = row_matches[-1].contents[offset]
+        return (
+            content.strip() if isinstance(content, str) else content.get_text().strip()
+        )
+    except Exception:
```

Comment:
Can you catch a specific exception here? It is especially dangerous to catch all errors since you are ignoring the error without logging it so you may miss an important error here.

Reply:
Narrowed the scope of the try block, and now catching `IndexError`.